cmd/crc/cmd/stop.go

```diff
@@ -50,13 +51,18 @@ func runStop(arguments []string) {
 		}
 		errors.ExitWithMessage(1, err.Error())
 	}
-	output.Out(commandResult.Success)
+	if commandResult.Success {
+		output.Out("CodeReady Containers instance stopped")
+	} else {
+		/* If we did not get an error, the status should be true */
+		logging.WarnF("Unexpected status %v", commandResult.Success)
+	}
 }
 
 func killVM(killConfig machine.PowerOffConfig) {
-	commandResult, err := machine.PowerOff(killConfig)
-	output.Out(commandResult.Success)
+	_, err := machine.PowerOff(killConfig)
 	if err != nil {
 		errors.ExitWithMessage(1, err.Error())
 	}
+	output.Out("CodeReady Containers instance killed")
```

Comment:
killed is a little bit harsh. This is internal naming.
I prefer to say: "powered off" as this is clearer to mean "it can be restarted" instead of "recreated"

Reply:
@robin-owen WDYT?

should we say "killed", "powered off" or "stopped", or possibly even "forcibly stopped" as this happens when the timeout of stopping has passed the 2mins mark.

Reply:
I want to make it clear that this was an unclean shutdown, so "forcibly stopped" would work for me.

Reply:
"Forcibly stopped" is the most clear in my opinion.